src/python/pants/backend/python/goals/run_pex_binary.py

```diff
@@ -31,6 +31,9 @@ async def create_pex_binary_run_request(
         built_pex = await Get(BuiltPackage, PexBinaryFieldSet, field_set)
         relpath = built_pex.artifacts[0].relpath
         assert relpath is not None
+        if field_set.layout.value != "zipapp":
```

Comment:
Lazy review, but the use of an enum in tests and a string here raises a red flag. Are the types right here? Probably, or tests would fail. So can the enum be used here instead?

Reply:
I'll use `.value` so it isn't stale over time